_release/attiny.yaml

```diff
@@ -1,2 +1,8 @@
 pi-wake-time: 20:00
 pi-sleep-time: 20:00
+
+voltages:
+  enable: false
+  no-battery: 10
+  low-battery: 369
+  full-battery: 455
```

Comment:
what are the units for these?

Reply:
These don't really have nice units. They are the analog voltage reading from the battery sense pin on the ATtiny. The analog read on the ATtiny is a 10 bit value from GND to +Ve.

Reply:
Could the config file have voltages which attiny-controller or the attiny firmware itself converts to the 10 bit values?

Reply:
For now I don't think this is will work as we don't have good data on how to correlate the battery sense value to a voltage. When we do we can have a voltage in the config file and attiny-controller can calculate what raw value that will be.

Reply:
Ok - makes sense.